src/main/java/seedu/duke/model/module/ModuleList.java

```diff
@@ -0,0 +1,52 @@
+package seedu.duke.model.module;
+
+import seedu.duke.DukeException;
+import seedu.duke.commons.core.Messages;
+import seedu.duke.model.module.exceptions.ModuleIndexException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ModuleList {
+    private List<Module> userModuleList; // a list of modules added by the user
+
+    public ModuleList() {
+        userModuleList = new ArrayList<>();
+    }
+
+    public int getSize() {
+        return userModuleList.size();
+    }
+
+    public Module getModule(int moduleIndex) throws DukeException {
```

Comment:
Consider changing this to `ModuleIndexException`.

Reply:
There is already another class named ModuleIndexException used by ModuleList!